api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -1833,3 +1837,93 @@ func validateDNS(dnsPolicy v1.DNSPolicy, dnsConfig *v1.PodDNSConfig) error {
 
 	return nil
 }
+
+func (c *AerospikeCluster) validateNetworkPolicy(namespace string) error {
+	networkPolicy := &c.Spec.AerospikeNetworkPolicy
+
+	annotations := c.Spec.PodSpec.AerospikeObjectMeta.Annotations
+	networks := annotations["k8s.v1.cni.cncf.io/networks"]
+	networkList := strings.Split(networks, ",")
+	networkSet := sets.NewString()
+
+	for ind := range networkList {
+		netName := strings.TrimSpace(networkList[ind])
+		namespacedName := strings.Split(netName, "/")
```

Comment:
This logic is similar to part of setNamespaceDefault function, can we re-use it somehow?

Reply:
Modified func and reused, however it'll result in 1 for loop to insert into set